http_2.rflx

```diff
@@ -0,0 +1,230 @@
+package HTTP_2 is
+
+   type Payload_Length is mod 2**24;
+
+   type Frame_Type is (
+      DATA => 0,
+      HEADERS => 1,
+      PRIORITY => 2,
+      RST_STREAM => 3,
+      SETTINGS => 4,
+      PUSH_PROMISE => 5,
+      PING => 6,
+      GOAWAY => 7,
+      WINDOW_UPDATE => 8,
+      CONTINUATION => 9
+   ) with Size => 8;
+
+   type Settings_Parameter is (
+      SETTINGS_HEADER_TABLE_SIZE => 16#01#,
+      SETTINGS_ENABLE_PUSH => 16#02#,
+      SETTINGS_MAX_CONCURRENT_STREAMS => 16#03#,
+      SETTINGS_INITIAL_WINDOW_SIZE => 16#04#,
+      SETTINGS_MAX_FRAME_SIZE => 16#05#,
+      SETTINGS_MAX_HEADER_LIST_SIZE => 16#06#
+   ) with Size => 16;
+
+   type Settings_Value_Generic is mod 2**32;
+   type Settings_Value_Enable_Push is range 0 .. 1 with Size => 32;
+   type Settings_Value_Initial_Window_Size is range 0 .. 2**31 - 1 with Size => 32;
+   type Settings_Value_Max_Frame_Size is range 2**14 .. 2**24-1 with Size => 32;
+
+   type Zero_Bit is range 0 .. 0 with Size => 1;
+   type Stream_ID is mod 2**31;
+   type Weight is mod 2**8;
+   type Error_Code is (
+      NO_ERROR => 16#00#,
+      PROTOCOL_ERROR => 16#01#,
+      INTERNAL_ERROR => 16#02#,
+      FLOW_CONTROL_ERROR => 16#03#,
+      SETTINGS_TIMEOUT => 16#04#,
+      STREAM_CLOSED => 16#05#,
+      FRAME_SIZE_ERROR => 16#06#,
+      REFUSED_STREAM => 16#07#,
+      CANCEL => 16#08#,
+      COMPRESSION_ERROR => 16#09#,
+      CONNECT_ERROR => 16#0A#,
+      ENHANCE_YOUR_CALM => 16#0B#,
+      INADEQUATE_SECURITY => 16#0C#,
+      HTTP_1_1_REQUIRED => 16#0D#
+   ) with Size => 32;
+
+   type Window_Size_Increment is range 1 .. 2**31 - 1 with Size => 31;
+   type Pad_Length is mod 2**8;
+
+   type Frame is
+      message
+         Payload_Length : Payload_Length
+            then Frame_Type
+               if Message'Size = (Payload_Length + 9) * 8;            -- Header length is always 9 octets
```

Comment:
Please don't arbitrarily indent comments, just use one whitespace before and after `--` for all comments.

Reply:
Chenged in e269eec

Reply:
> Please don't arbitrarily indent comments, just use one whitespace before and after `--` __for all comments__.

Reply:
Didn't see the second comment. Removed it in 7d43ab6